.github/workflows/pr_verification.yml

```diff
@@ -92,3 +92,26 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           arguments: test
+
+  bugzilla_check:
+    name: bugzilla reference check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.BUGZILLA_API_TOKEN }}"
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
this checks runs on "ubuntu-latest", which is currently 22.04 LTS [1]: this means it has python 3.10 has default version, so you don't need to install a custom python; simply install `python3` and `python3-pip`


[1] https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

Reply:
Good catch! Thanks for linking the github hosted runner docs. I added an 'Install packages' step where I install the `python3` and `python3-pip` package. I also updated the bugzilla_reference_check.py to use python3 instead of python.